source/includes/_processing.md

```diff
@@ -0,0 +1,314 @@
+        "new": "World!"
+    }
+}]
+```
+
+The payload lists the processing stack of the dataset. Please note that it reads in the order in which processors are applied.
+
+
+### Returns
+The list of [processor objects](#the-processor-object).
+
+## Append a new processor to a dataset
+
+This endpoint is for creating a new processor for the dataset. The processor will be appended to the end of the processing stack.
+
+> Definition
+
+```HTTP
+POST https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/
+```
+
+> Example request
+
+```HTTP
+curl -XPOST https://yourdomain.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/
+    -u username:password \
+    -d '{"name": "string_extractor", "args": {"field": "text_field", "regexp": "(?P<planet>^World)"}'
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_newnew",
+    "name": "string_extractor",
+    "args": {
+        "field": "text_field",
+        "regexp": "(?P<planet>^World)"
+    }
+}
+```
+
+The endpoint takes a processor object (without its processor_uid, which will be automatically generated), creates a processor with the provided configuration and appends it at the end of the processing chain.
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+`name` <br> *string* | name of the processor
+`args` <br> *object* | processor-dependent parameters
+
+
+### Returns
+The newly created [processor object](#the-processor-object).
+
+
+## Retrieve informations about a processor
+
+This endpoint is meant to retrieve a processor object in a dataset processing stack from its uid.
+
+> Definition
+
+```HTTP
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XGET https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_newnew
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_newnew",
+    "name": "string_extractor",
+    "args": {
+        "field": "text_field",
+        "regexp": "(?P<planet>^World)"
+    }
+}
+```
+
+### Returns
+The requested [processor object](#the-processor-object).
+
+## Update a processor
+
+This endpoint is meant to update a processor in place within the processing stack of a dataset.
+Explanation of the thing
+
+> Definition
+
+```
+PUT https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_abcdef
+    -u username:password \
+    -d '{"name":"regexp_replace", "args": {"field": "text_field", "regexp": ".*", "new": "Bonjour Monde"}'
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_abcdef", 
+    "name":"regexp_replace",
+    "args": {
+        "field": "text_field",
+        "regexp": ".*",
+        "new": "Hello World"
```

Comment:
I think the response should take the update into account, no?

Reply:
:+1: